docs/content/how-to/mce/Dual/hostedcluster/hostedcluster.md

```diff
@@ -0,0 +1,248 @@
+In this section we will focus in all the related objects which they will be necessary to achieve a Disconnected Hosted Cluster deployment.
+
+**Premises**:
+
+- HostedCluster Name: `hosted-dual`
+- HostedCluster Namespace: `clusters`
+- Disconnected: `true`
+- Network Stack: `Dual`
+
+## Openshift Objects
+
+### Namespaces
+
+The operator in a usual situation will be in charge of create the HCP (HostedControlPlane) namespace, but in this case we want to include all the objects prior the operator to reconcicle over the HostedCluster object. This way when the operator begins the reconcilliation, it will find all the objects in place.
```

Comment:
```suggestion
The hypershift operator is usually in charge of creating the HCP (HostedControlPlane) namespace, but in this case, we want to include all the objects prior to the hypershift operator reconciling over the HostedCluster object. This way when the hypershift operator begins the reconciliation, it will find all the objects in place.
```